pkgs/common-updater/scripts/update-source-version

```diff
@@ -75,11 +79,39 @@ if [[ -z "$versionKey" ]]; then
     versionKey=version
 fi
 
+# Allow finding packages among flake outputs in repos using flake-compat.
+pname=$(nix-instantiate $systemArg --eval --strict -A "$attr.name" || echo)
+if [[ -z "$pname" ]]; then
```

Comment:
New test does not require quotes.

Reply:
I am aware but I prefer to use quotes everywhere to normalize their use.

Reply:
plus when running shellcheck etc, it limits false positives.